packages/core/jest.config.js

```diff
@@ -6,6 +6,11 @@ module.exports = {
   testEnvironment: 'node',
   testMatch: ["**/?(*.)+(test).[jt]s?(x)"],
   clearMocks: true,
+  setupFilesAfterEnv: ['./jest.setup.js'],
+  projects: [
+    "<rootDir>",
+    "<rootDir>/../core-integration-tests"
```

Comment:
running "yarn test" and so forth in core automatically runs integration tests (yes, jest has its own support for monorepos!)